odev/setup/symlink.py

```diff
@@ -33,11 +37,12 @@ def setup(odev: Odev) -> None:
 
         if console.confirm("Would you like to overwrite it?"):
             logger.debug(f"Removing symlink path {link_path}")
-            link_path.unlink(missing_ok=True)
+            bash.execute(f"rm {link_path}", sudo=True)
```

Comment:
Don't you have a risk to delete ~/.local/bin folder now ? ( local_path can be one of the path from $PATH. and Odev is only added to the path if there is no match , meaning that if the folder is a symlink it will be deleted ...